packages/components/addon/components/hds/alert/index.js

```diff
@@ -0,0 +1,54 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+export default class HdsAlertIndexComponent extends Component {
+  /**
+   * Get the class names to apply to the component.
+   * @method Alert#classNames
+   * @return {string} The "class" attribute to apply to the component.
+   */
+  // "hds-alert"
+  get classNames() {
```

Comment:
[nitpick] we move this specific method at the end for every component, so it's easy to find

Reply:
sounds good, done https://github.com/hashicorp/design-system/pull/92/commits/6d3e499f89d08ae47b14e10caf517561162b38f0